tests/network/sriov.go

```diff
@@ -697,16 +697,21 @@ func createAndWaitVMI(vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstanc
 		return nil, err
 	}
 
-	return waitVMI(virtClient, vmi)
+	return waitVMI(vmi)
 }
 
-func waitVMI(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstance, error) {
+func waitVMI(vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstance, error) {
```

Comment:
Can you please add the reasoning for this change in the commit message?

Reply:
Do I really need to explain why it is simpler?
As I need to add other changes, I do not mind much, but this one seems trivial enough to me.
Done.

Reply:
I did not know that getting the client more then once has little cost, this is the reason I've asked.
Thanks for the explanation.